topologytest/peer_test.go

```diff
@@ -411,40 +411,34 @@ func TestPeerImplementation(t *testing.T) {
 			require.Equal(t, updateVersion.docMeta, roundtripGetVersion)
 			require.JSONEq(t, string(updateBody), string(base.MustJSONMarshal(t, roundtripGetbody)))
 
-			if peer.Type() == PeerTypeCouchbaseLite {
-				// CBG-4432 Couchbase Lite peer does not support deletion yet
-				return
-			}
-
 			// Delete
 			deleteVersion := peer.DeleteDocument(collectionName, docID)
 			require.NotEmpty(t, deleteVersion.CV(t))
 			require.NotEqual(t, deleteVersion.CV(t), updateVersion.docMeta.CV(t))
 			require.NotEqual(t, deleteVersion.CV(t), createVersion.docMeta.CV(t))
-			if tc.peerOption.Type == PeerTypeCouchbaseServer {
-				require.Empty(t, deleteVersion.RevTreeID)
-			} else {
+			if tc.peerOption.Type == PeerTypeSyncGateway {
```

Comment:
I saw this change elsewhere previously - and now that CBL is enabled, SG is the only peer that is writing RevTreeIDs.

Reply:
This test was just to make sure the testing code works correctly, this seems right.